components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -2731,7 +2722,12 @@ def groupSummary(self, gid=None, exclude_self=False):
         :return:    {'leaders': list :class:`ExperimenterWrapper`,
                      'colleagues': list :class:`ExperimenterWrapper`}
         :rtype:     dict
+        
+        ** Deprecated ** Use :meth:`ExperimenterGroupWrapper.groupSummary`.
```

Comment:
@chris-allan @will-moore I deprecated getSummary on BlitzObjectWrapper object as it has not sense to me. It makes more send it to have on ExperimenterGroupWrapper https://github.com/openmicroscopy/openmicroscopy/pull/3478/files#diff-e4feecea30b2ee097639cbb7c9171fedR5283